Navigation/Profile/ProfileViewController.swift

```diff
@@ -7,48 +7,145 @@
 
 import UIKit
 
-class ProfileViewController: UIViewController {
+class ProfileViewContoller: UIViewController {
+    
+    private let posts: [String] = [
+        "Books",
+        "Movies",
+        "Show",
+        "Science"
+    ]
     
     private let profileHeaderView = ProfileHeaderView()
     
-    lazy var button: UIButton = {
-        let button = UIButton(type: .system)
-        button.translatesAutoresizingMaskIntoConstraints = false
-        button.backgroundColor = .systemOrange
-        button.setTitle("Tap me", for: .normal)
-        button.setTitleColor(.black, for: .normal)
-//        button.layer.cornerRadius = 4
-        return button
+    private lazy var tableView: UITableView = {
+        let tableView = UITableView(frame: .zero, style: .grouped)
+        
+        /// Register a cell. Makes a cell visible.
+        tableView.register(PostTableViewCell.self, forCellReuseIdentifier: String(describing: PostTableViewCell.self))
+        tableView.dataSource = self
+        tableView.delegate = self
+        /// Delete scroll indicator from right side.
+        tableView.showsVerticalScrollIndicator = true
+        return tableView
     }()
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        setUpProfileView()
+        view.addSubview(profileHeaderView)
+        
+        setUpProfileViewController()
     }
     
-    func setUpProfileView() {
-        view.backgroundColor = UIColor.lightGray
+    private func setUpProfileViewController() {
+        [tableView].forEach {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+            view.addSubview($0)
+        }
         
-        navigationController?.navigationBar.prefersLargeTitles = false
-        navigationItem.title = "Profile"
+        NSLayoutConstraint.activate([
+            tableView.topAnchor.constraint(equalTo: view.topAnchor),
+            tableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            
+            profileHeaderView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
```

Comment:
You don't have profileHeaderView added to hierarchy, so these constraints are useless